### PR TITLE
All calendar months now same size + started adding user input functionality

### PR DESCRIPTION
**Pull request by myothra7777**

### CODE REVIEW

File: frontend/components/Calendar.jsx [hunk 1]
```diff
@@ -1,25 +1,49 @@
-import {Calendar} from 'react-native-calendars';
-import {View, Text} from 'react-native';
+import { Calendar } from 'react-native-calendars';
+import { View /* ,  Dimensions */ } from 'react-native';
 import React from 'react';
 
+// const windowHeight = Dimensions.get('window').height;
```
Comment (Cessna210) [inline frontend/components/Calendar.jsx:5]: Remove commented-out code

File: frontend/components/Calendar.jsx [hunk 2]
```diff
@@ -1,25 +1,49 @@
...
+
+const firstDayOfYear = new Date(new Date().getFullYear(), 0, 1);
+const lastDayOfYear = new Date(new Date().getFullYear(), 11, 31);
+
+const datesMarked = {
+  /*   '2020-09-25': {
```
Comment (Cessna210) [inline frontend/components/Calendar.jsx:11]: Remove commented-out code

File: frontend/components/Calendar.jsx [hunk 3]
```diff
@@ -1,25 +1,49 @@
...
-      <View>
+      <View style={{ flex: 1 }}>
         <Calendar
-          style={{borderWidth: 5, borderColor: 'green', height: 400}}
+          style={{ borderWidth: 5, borderColor: 'green' }}
           theme={{
+            'stylesheet.day.basic': {
+              base: {
+                width: '50%',
+                // height: windowHeight / 9,
```
Comment (Cessna210) [inline frontend/components/Calendar.jsx:31]: Remove commented-out code

File: frontend/components/Calendar.jsx [hunk 4]
```diff
@@ -1,25 +1,49 @@
...
+
+const datesMarked = {
+  /*   '2020-09-25': {
+      selected: true,
+      marked: true,
+      selectedColor: 'green',
+    },
+    '2020-09-29': {marked: true},
+    '2020-09-30': {marked: true, dotColor: 'red', activeOpacity: 0},
+    '2020-09-28': {disabled: true, disableTouchEvent: true}, */
```
Comment (Neodusk) [inline frontend/components/Calendar.jsx:18]: Please remove un-used code

**Review by Neodusk — CHANGES_REQUESTED** [review]

**Review by maklh899 — APPROVED** [review]

**Review by Neodusk — APPROVED** [review]

**Pull request merged**